applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -71,14 +75,49 @@ namespace Kratos
     //************************************************************************************
     //************************************************************************************
 
-    void BaseSolidElement::FinalizeSolutionStep( ProcessInfo& CurrentProcessInfo )
+    void BaseSolidElement::FinalizeSolutionStep( ProcessInfo& rCurrentProcessInfo )
```

Comment:
Just to add my comment: The name of argument does not affect the overriding. Only the type of them is important.